packages/react-hook-form/src/useFormGql.tsx

```diff
@@ -76,6 +80,7 @@ export function useFormGql<Q, V extends FieldValues>(
       }
       // if (variables === false) onInvalid?.(formValues, event)
 
+      submittedVariables.current = variables
```

Comment:
Should we also empty the value when a user resubmits? I would expect the value to be set to undefined on line 72?